File: week 1 - C/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;

    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 | height > 8); //Take user input

    for (int stair = 0; stair < height; stair++) //Loop stair by stair
    {
        for (int i = 1; i < (height - stair); i++) //Left side gap
        {
            printf(" ");
        }

        for (int i = 0; i <= stair; i++) //Left side bricks
        {
            printf("#");
        }

        printf("  "); //Gap between left and right side bricks

        for (int i = 0; i <= stair; i++) //Right side bricks
        {
            printf("#");
        }

        printf("\n"); //New line for next stair
    }
}

File: CS50: Introduction to Computer Science/week 6 - Python/mario.py
from cs50 import get_int

while True:
    height = get_int("Height: \n")
    if height > 0 and height < 9:
        break

for stair in range(1, height+1):
    print("{0}{1}{2}{1}".format((height - stair) * " ", stair * "#", "  "))
